client.go

```diff
@@ -26,11 +26,16 @@ import (
 	"strings"
 	"time"
 
+	internal "github.com/aerospike/aerospike-client-go/v8/internal/version"
 	"github.com/aerospike/aerospike-client-go/v8/logger"
 	"github.com/aerospike/aerospike-client-go/v8/types"
 )
 
-const unreachable = "UNREACHABLE"
+const (
+	unreachable        = "UNREACHABLE"
+	no_password        = "nopassword"
```

Comment:
These are not namespace-wide constant. Should be moved inside the one function they are used in. Snake-case values are also not idiomatic to Go, should use camelCase.